cellar/cellar_extractor/operative_extraction.py

```diff
@@ -0,0 +1,399 @@
+        """
+        website=requests.get(f"https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A{self.celex}&from=EN").text
+        parser=BeautifulSoup(website,'lxml')
+        div=parser.find_all('table') # Find all tables tag from the website
+        one=[]
+        for divs in div:
+            table=divs.find('table') # Find each nested table within the table
+            if  table!=None:
+                p=table.find_all('p',class_="coj-normal") # Find all p under the nested table with the coj-normal class
+                for x in p:
+                    span=x.find_all('span',class_="coj-bold")# Span class of coj-bold under the p tag
+                    for y in span:
+                        if x!=None and y!=None:
+                 
+                            one.append(y.text)#append text from span onto a list
+        return one            
+                
+
+    def html_page_structure_two(self)->list:
+        """
+         This function retreives operative part from documents of the respected celex id's . This function scrapes/parse the operative part from a paragraph
+         (p) structure . The relevant text lies inside the normal class of the p tag which comes after the keyword operative of the previous span tag.
+        """
+        website=requests.get(f"https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A{self.celex}&from=EN").text
+        parser=BeautifulSoup(website,'lxml')
+        p=parser.find_all('p')
+        two=[]
+        for para in p:
+        
+            span=para.find('span')
+            if span!=None:
+            
+                if "operative" in span.text.lower():
+                    normal=span.find_all_next('p',class_="normal")
+                    for op in normal:
+                    
+                        two.append(op.text)
+        return two          
+        
+    def structure_three(self)->list:
+        """
+         This function retreives operative part from documents of the respected celex id's . This function scrapes/parse the operative part from a nested
+         table structure . The relevant text lies inside the coj-bold class of the span tag.
+        """
+        website=requests.get(f"https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A{self.celex}&from=EN").text
+        parser=BeautifulSoup(website,'lxml')
+        table=parser.find_all('table')
+        three=[]
+        for tables in table:
+            interior=tables.find_all('table')
+            for interiors in interior:
+                if interiors!=None:
+                    p=interiors.find_all('p',class_="coj-normal")
+                    for x in p:
+                        span=x.find_all('span',class_="coj-bold")
+                        for y in span:
+                            if x!=None and y!=None:
+                
+                                three.append(y.text)
+        return three            
+
+
```

Comment:
You need to clean up such that this sticks to PEP8 standard. Unnecessary new lines and indentation in code. Please lint your code.